tests/e2e/runner.py

```diff
@@ -36,18 +37,17 @@ def load_args():
 
 def load_workloads(root_directory):
     workloads = []
-    for file in Path(root_directory).rglob('*.yaml'):
+    for file in Path(root_directory).rglob("*.yaml"):
```

Comment:
@jbajic you mentioned yesterday that `.yaml` is the wrong extension. Shall we create a task for it that replaces the extension with the proper one?

Reply:
I think its fine the way it is, it is not a big deal :smile: